packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.tsx

```diff
@@ -577,7 +574,7 @@ const DateRangeCalendar = React.forwardRef(function DateRangeCalendar(
       ownerState={ownerState}
       {...other}
     >
-      <Watermark packageName="x-date-pickers-pro" releaseInfo={releaseInfo} />
+      <Watermark packageName="x-date-pickers-pro" releaseInfo="__RELEASE_INFO__" />
```

Comment:
```suggestion
      <Watermark packageName="x-date-pickers-pro" releaseInfo={releaseInfo} />
```

Better assign it to the variable to have a stable object reference